redpanda/templates/_helpers.tpl

```diff
@@ -31,12 +31,7 @@ If release name contains chart name it will be used as a full name.
 {{- if .Values.fullnameOverride }}
 {{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
 {{- else }}
-{{- $name := default .Chart.Name .Values.nameOverride }}
-{{- if contains $name .Release.Name }}
-{{- .Release.Name | trunc 63 | trimSuffix "-" }}
-{{- else }}
-{{- printf "%s" $name | trunc 63 | trimSuffix "-" }}
-{{- end }}
+{{- printf "%s" .Release.Name | trunc 63 | trimSuffix "-" }}
```

Comment:
Does this change invalidate the [comment on line 28](https://github.com/vectorizedio/helm-charts/pull/18/commits/6136551fbde8b001a3d38b6339b6f43a3c50547d#diff-407fdf464d0cee923155f3378574ffbd8f9fbedacd1c0c3a37fcad7d2d438071R28)?:

> If release name contains chart name it will be used as a full name

Reply:
Yes it does, I removed that comment.